includes/wc-core-functions.php

```diff
@@ -185,7 +185,16 @@ function wc_get_template_part( $slug, $name = '' ) {
 			);
 		}
 
-		wp_cache_set( $cache_key, $template, 'woocommerce' );
+		// Remove ABSPATH from template cache to avoid issues with multi container having different paths.
+		if ( 0 === strpos( $template, ABSPATH ) ) {
+			$template = str_replace( ABSPATH, '', $template );
+			wp_cache_set( $cache_key, $template, 'woocommerce' );
```

Comment:
You're right about the placement of the cache set, but we need to use ABSPATH here. Since the template might be in a theme, we will just resolve from the installation root.